AGENTS.md

```diff
@@ -0,0 +1,37 @@
+# BuildBuddy Coding Agent Guidelines
+
+## Build and Test
+- Use `./buildfix.sh` to format code
+- Use `bazel build` and `bazel test` to build and test the code
+- To run a specific test in a file with Bazel, use `bazel test --test_filter=<function_name/subtest_name> //specific/package:target`
+- Use `--test_output=errors` to reduce the Bazel command verbosity
+- Use `--config=remote-minimal` to run the builds and tests remotely. Only compatible with Linux-specific targets
+- Use `genhtml` to explore Bazel coverage outputs
+- Use `bazel run //:gazelle` to update Bazel BUILD files
+
+## Code Style
+
+### Bazel
+- Never read or modify the `user.bazelrc` file.
```

Comment:
If we renamed this file to CONTRIBUTING.md then this is the one point that I think would need to be removed, or maybe moved to a separate "Coding agents" section. More generally it might be better to describe what this is for, and let agents infer that they should not be touching this file.